### DISCUSSION TITLE
Updated and Fixed Examples

### DISCUSSION THREAD

**Discussion by iluvcapra** [iluvcapra/py-ptsl#3] [General]
@patrickcusack @will-riley @chigkim @AdamVenn

Hi All!

I did a bunch of work this weekend and it seems to have broken the examples, I fixed them tonight so they should all be working again.

I'm curious to hear if any of you have given the module a try and what your experience has been installing or working with it. Drop me a response here if you have anything to share.

**Comment by pneyrinck** [comment]
Hi. Here is the output I see when I try to install. Is this an issue or is it an error on my end?

(venv) pauls-mbp-m1:py-ptsl pneyrinck$ python -m pip install .
Processing /Users/pneyrinck/Code/py-ptsl
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [10 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/Users/pneyrinck/Code/py-ptsl/setup.py", line 4, in <module>
          from ptsl import __VERSION__
        File "/Users/pneyrinck/Code/py-ptsl/ptsl/__init__.py", line 4, in <module>
          from .client import Client
        File "/Users/pneyrinck/Code/py-ptsl/ptsl/client.py", line 8, in <module>
          from google.protobuf import json_format
      ModuleNotFoundError: No module named 'google'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

**Reply by iluvcapra** [reply]
Sorry saw the PM first. You should pull the latest head, this got fixed this weekend.

**Comment by AdamVenn** [comment]
Awesome! Gonna make a little time to try this out this week!

**Comment by AdamVenn** [comment]
Sorry, I was not able to make any more time than was enough to update Pycharm and clone the repo. :-( Next week it is.

**Comment by will-riley** [comment]
Just started to check out the examples, all working except for make_session.py, which wanted a trailing slash in the path, then fine. I'll dig in further as the current dub tapers off.
